internal/node/node_loader.js

```diff
@@ -182,30 +191,35 @@ function resolveRunfiles(...pathSegments) {
 
     let maybe = resolveManifestFile(runfilesEntry);
     if (maybe) {
-      if (DEBUG) console.error("node_loader: resolved manifest file", maybe);
+      if (DEBUG)
+        console.error("node_loader: resolved manifest file", maybe);
       return maybe;
     }
 
     maybe = resolveManifestDirectory(runfilesEntry);
     if (maybe) {
-      if (DEBUG) console.error("node_loader: resolved manifest directory", maybe);
+      if (DEBUG)
+        console.error("node_loader: resolved manifest directory", maybe);
       return maybe;
     }
   } else {
     let maybe = loadAsFileSync(defaultPath);
     if (maybe) {
-      if (DEBUG) console.error("node_loader: resolved file", maybe);
+      if (DEBUG)
+        console.error("node_loader: resolved file", maybe);
       return maybe;
     }
 
     maybe = loadAsDirectorySync(defaultPath);
     if (maybe) {
-      if (DEBUG) console.error("node_loader: resolved directory", maybe);
+      if (DEBUG)
+        console.error("node_loader: resolved directory", maybe);
       return maybe;
     }
   }
 
-  if (DEBUG) console.error("node_loader: resolved to default path", defaultPath);
+  if (DEBUG)
```

Comment:
I think the linter should complain that this is no longer on one line and should now have braces.
don't need to fix now...

Reply:
right. I forgot to add a .clang-format file so its on the default format. I'll set it to ```Language:        JavaScript
BasedOnStyle:    Google
ColumnLimit:     100
```
I think that style adds braces